code/FBXConverter.cpp

```diff
@@ -708,10 +719,13 @@ void Converter::ConvertCamera( const Model& model, const Camera& cam )
     out_camera->mName.Set( FixNodeName( model.Name() ) );
 
     out_camera->mAspect = cam.AspectWidth() / cam.AspectHeight();
-    out_camera->mPosition = cam.Position();
-    out_camera->mUp = cam.UpVector();
-    out_camera->mLookAt = cam.InterestPosition() - out_camera->mPosition;
+    //cameras are defined along positive x direction
+    out_camera->mPosition = aiVector3D(0.0f);
+    out_camera->mLookAt = aiVector3D(1.0f, 0.0f, 0.0f);
+    out_camera->mUp = aiVector3D(0.0f, 1.0f, 0.0f);
     out_camera->mHorizontalFOV = AI_DEG_TO_RAD( cam.FieldOfView() );
```

Comment:
All the necessary transform information is in the cameras node. So Fbx stores the cameras transformation in the scene graph as well as in the camera settings. This means you can either use one or the other but if you use both then the transform ends up getting applied twice resulting in incorrect values. Assimp already adds the required transformation info to the cameras node so the transform in the camera settings can be ignored.

I tested this with files exported from both blender and maya.